tests/test_pandas.py

```diff
@@ -130,6 +130,49 @@ def test_csv():
     assert_equal(df.columns.tolist(), df2.columns.tolist())
     assert_equal(df.dtypes.tolist(), df2.dtypes.tolist())
 
+    with NamedTemporaryFile(suffix='.csv', mode='w+') as f:
+        df.to_csv(f, index=False, columns=['name', 'act'])
```

Comment:
Pandas 0.13 uses `cols` kwarg. https://pandas.pydata.org/pandas-docs/version/0.13/generated/pandas.DataFrame.to_csv.html We need to fix that bug.

Reply:
Fun fact: it uses `columns` in `to_html`: https://pandas.pydata.org/pandas-docs/version/0.13/generated/pandas.DataFrame.to_html.html#pandas.DataFrame.to_html